### ISSUE TITLE
How to use an alternative theme?

### ISSUE THREAD

**Issue by BoPeng** [tscanlin/tocbot#119]
The word `theme` sounds like there are multiple themes to choose from and I would prefer the tocify-like theme to the default theme. Is there anyway to configure tocbot not to use the default theme, and/or define and use an alternative theme?

**Comment by tscanlin** [comment]
yes you may create your own custom theme using plain CSS. Feel free to style it as you wish.

**Comment by BoPeng** [comment]
Do I have to override `.toc-link::before` etc after I load 
```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot.css">
```

Frankly I know how to override but do not know how to remove `.toc-link::before`.

**Comment by tscanlin** [comment]
Its the order the CSS appears on the page so overrides must come after tocbot default css.

**Comment by BoPeng** [comment]
Thanks. Basically I was looking for something like

```
https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot-core.css
```
so that I do not have to remove `.toc-link::before` from `tocbot.css`.

**Comment by BoPeng** [comment]
OK, with the css #120, ID fix #122, and tocbot configurations defined in https://github.com/vatlab/sos-notebook/blob/master/src/sos_notebook/templates/parts/toc.tpl , I have achieved the tocify-like TOC in pages such as https://vatlab.github.io/sos-docs/doc/user_guide/sos_in_notebook.html . Hopefully this could help some other users of tocbot.
